src/popup/components/Modals/MultisigVaultCreate.vue

```diff
@@ -24,7 +24,7 @@
       >
         <FormSelect
           v-if="index === 0"
-          v-model.trim="signer.address"
+          v-model="signer.address"
```

Comment:
Why have you decided to remove `.trim`?

Reply:
> Why have you decided to remove `.trim`?

Address names can have spaces in them, we cannot use .trim here, otherwise the user will not be able to search correctly for the address names

Reply:
I can revert it back if you'd like. In this exact instance it doesn't matter since the address is filled automatically via the "Signer's account address" modal, and not typed in.
In other instances it was necessary to remove the .trim to make the searching work properly, since Address Names can have spaces in them.